src/drivers/uavcan/uavcan_drivers/stm32h7/driver/src/uc_stm32h7_can.cpp

```diff
@@ -147,6 +147,12 @@ inline void handleTxInterrupt(uavcan::uint8_t iface_index)
 		}
 
 		ifaces[iface_index]->handleTxInterrupt(utc_usec);
+
+	} else if ((fdcan::Can[iface_index]->IR & FDCAN_IR_BO) > 0) {
```

Comment:
```suggestion
	} else if ((fdcan::Can[iface_index]->IR & FDCAN_IR_BO) != 0) {
```
Mixing bit ops and magnitude testing is a bit odd.